weatherman.py

```diff
@@ -0,0 +1,246 @@
+            average_highest_temp += __weather__["Murree"][year][month][day]["Max TemperatureC"]
+        return average_highest_temp//no_of_months
+
+    @staticmethod
+    def average_lowest_temperature(year, month):
+        average_lowest_temp = 0
+        no_of_months = len(__weather__["Murree"][year][month].keys())
+        for day in __weather__["Murree"][year][month].keys():
+            average_lowest_temp += __weather__["Murree"][year][month][day]["Min TemperatureC"]
+        return average_lowest_temp//no_of_months
+
+    @staticmethod
+    def average_mean_humidity(year, month):
+        avg_mean_humidity = 0
+        no_of_months = len(__weather__["Murree"][year][month].keys())
+        for day in __weather__["Murree"][year][month].keys():
+            avg_mean_humidity += __weather__["Murree"][year][month][day]["Mean Humidity"]
+        return avg_mean_humidity//no_of_months
+
+    @staticmethod
+    def maximum_temperature_on_day(year, month, day):
+        return __weather__["Murree"][year][month][day]["Max TemperatureC"]
+
+    @staticmethod
+    def minimum_temperature_on_day(year, month, day):
+        return __weather__["Murree"][year][month][day]["Min TemperatureC"]
+
+
+class Report:
+
+    @staticmethod
+    def maximum_temperature(year):
+        temperature, day, month = Computations.maximum_temperature(year)
+        print("Highest: " + str(temperature) + "C on " + str(month) + " " + str(day))
+
+    @staticmethod
+    def minimum_temperature(year):
+        temperature, day, month = Computations.minimum_temperature(year)
+        print("Lowest: " + str(temperature) + "C on " + str(month) + " " + str(day))
+
+    @staticmethod
+    def maximum_humidity(year):
+        humidity, month, day = Computations.maximum_humidity(year)
+        print("Humidity: " + str(humidity) + "% on " + str(month) + " " + str(day))
+
+    @staticmethod
+    def average_highest_temperature(year, month):
+        average = Computations.average_highest_temperature(year, month)
+        print("Highest Average: " + str(average) + "C")
+
+    @staticmethod
+    def average_lowest_temperature(year, month):
+        average = Computations.average_lowest_temperature(year, month)
+        print("Average Mean Humidity: " + str(average) + "%")
+
+    @staticmethod
+    def average_mean_humidity(year, month):
+        average = Computations.average_mean_humidity(year, month)
+        print("Lowest Average: " + str(average) + "C")
+
+    @staticmethod
+    def generate_barchart(temp, is_highest):
+        if is_highest:
+            return "\033[91m {}\033[00m".format("+" * temp)
+        return "\033[94m {}\033[00m".format("+" * temp)
+
+    @staticmethod
+    def maxmin_temperature_on_days(year, month):
+        for day in __weather__["Murree"][year][month]:
+            temp = Computations.maximum_temperature_on_day(year, month, day)
+            temp_ = Computations.minimum_temperature_on_day(year, month, day)
```

Comment:
What does `temp` and `temp_` means? Rename it to something sensible and meaningful.